handlers/line/line.go

```diff
@@ -300,19 +320,29 @@ func (h *handler) Send(ctx context.Context, msg courier.Msg, logger *courier.Cha
 			}
 		}
 	}
+
+	attachments, err := handlers.ResolveAttachments(ctx, h.Backend(), msg.Attachments(), mediaSupport, true)
```

Comment:
`allowURLOnly` should be false otherwise this can return attachments that don't have associated media objects

Reply:
Added a test PDF media and enforced that